duradmin/src/main/webapp/WEB-INF/config/webmvc-config.xml

```diff
@@ -30,6 +30,8 @@
   </context:component-scan>
   <mvc:annotation-driven/>
 
+  <mvc:resources mapping="/webjars/**" location="/webjars/"/>
```

Comment:
Similar to the comment above - I may be missing something, but I'm not seeing what resources are being pulled in here.